admin/dashboard/catalog.md

```diff
@@ -1,15 +1,3 @@
 # Catalog
 
-## What is catalog revision control?
-
-Revision control in Reaction adds the ability for admin users to review, stage, and revert changes to products. When enabled, any changes to a product will be saved as a draft, with the original product remaining unchanged. Upon publishing changes to that draft, the publicly visibly product will then have those changes applied.
-
-At the moment revision control is merely used to stage a single upcoming update.
-
-## Enabling and disabling revision control
-
-In the dashboard click <i class="font-icon fa fa-book"></i> **Catalog**, where you may toggle revision control `on`, or `off`.
-
-Currently, turning off revision will give you the legacy version of the product detail page.
-
-![](/assets/admin-catalog-revision-settings.png "Catalog Revision Settings")
+> _Deprecation warning:_ Revision Control has been deprecated from Reaction as of 1.13. The legacy revision control system has been deprecated in favor of the new Catalog collection for overall product gird performance improvements and simplifed product publication logic. Read more about the changes in the CHANGELOG and [here](https://github.com/reactioncommerce/reaction/pull/4238).
```

Comment:
The link [here] should say where "here" is. Maybe `[pull request]` or something else that makes sense?

Read more about the changes in the CHANGELOG and in this [pull request](https://github.com/reactioncommerce/reaction/pull/4238).

Reply:
Also might as well link to the changelog too.